css/build.css

```diff
@@ -125,4 +125,12 @@ script {
   -webkit-transform: none;
   -ms-transform: none;
   transform: none;
-}
\ No newline at end of file
+}
+
+.panel.panel-default.focus-outline {
```

Comment:
Under what scenario will the `focus-outline` be missing?

Reply:
@tonytlwu So far I can't see such scenarios. Should I remove `focus-outline` from the selector?

Reply:
Yes please

Reply:
@tonytlwu removed it.